google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/BigQueryRpc.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.bigquery.spi;
+package com.google.cloud.bigquery.spi.v2;
```

Comment:
We don't need to move the `*RpcFactory` interfaces because they are actually empty interfaces, and aren't tied to a specific major version or transport.